docs/api/api_blueprint.apib

```diff
@@ -11548,7 +11548,7 @@ Delete a single session. **(Minimum Co-Organizer Access)**
     + sort (optional, string, `created-at`) - Sort the resources according to the given attribute in ascending order. Append '-' to sort in descending order.
     + filter (optional, string, `[]`) - Filter according to the flask-rest-jsonapi filtering system. Please refer: http://flask-rest-jsonapi.readthedocs.io/en/latest/filtering.html for more.
 
-### List Sessions [GET]
+### List Sessions under an Event [GET]
```

Comment:
If you're changing this here. You also have to change `@hooks.before("Sessions > List Sessions under an Event > List Sessions")` to `@hooks.before("Sessions > List Sessions under an Event > List Sessions under an Event")` [here](https://github.com/fossasia/open-event-server/blob/1e3f4545a315f4e67022f8856d7c2ad2d6227a7f/tests/hook_main.py#L1065)

Similarly make changes elsewhere.

Reply:
OK. I'm on it :+1: 
And also please help me with this issue https://github.com/fossasia/open-event-server/issues/5464
It's keeping me back from working